Sources/Basic/OutputByteStream.swift

```diff
@@ -52,20 +138,19 @@ public class OutputByteStream: TextOutputStream {
     private static let bufferSize = 1024
 
     /// Queue to protect mutating operation.
-    private let queue = DispatchQueue(label: "org.swift.swiftpm.basic.stream")
+    fileprivate let queue = DispatchQueue(label: "org.swift.swiftpm.basic.stream")
 
-    init() {
+    internal init() {
```

Comment:
We don't really write `internal` explicitly in SwiftPM codebase.

Reply:
Will fix.